### PR TITLE
Bulk update bug fixed

### PR DESCRIPTION
**Pull request by md-rafeek**
## Description
36103 - Access-Point Configuration: Need support for bulk operations
42969 - Providing string value for LED Brightness level parameter results in module error

## Type of Change
- [x] Bug fix
- [x] New feature
- [ ] Breaking change
- [x] Documentation update

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] All the sanity checks have been completed and the sanity test cases have been executed

## Ansible Best Practices
- [x] Tasks are idempotent (can be run multiple times without changing state)
- [ ] Variables and secrets are handled securely (e.g., using `ansible-vault` or environment variables)
- [x] Playbooks are modular and reusable
- [x] Handlers are used for actions that need to run on change

## Documentation
- [x] All options and parameters are documented clearly.
- [x] Examples are provided and tested.
- [x] Notes and limitations are clearly stated.

### CODE REVIEW

File: plugins/modules/accesspoint_workflow_manager.py [hunk 1]
```diff
@@ -2584,6 +2586,32 @@ def camel_to_snake_case(self, config):
...
+        """
+        Bulk access point changes collect each output update in the response.
+
+        Parameters:
+            self (dict): A dictionary used to collect the execution results.
+
+        Returns:
+            dict: A dictionary containing the result of the access point update response.
+        """
+        each_result = {"changed": self.result["changed"],
```
Comment (madhansansel) [inline plugins/modules/accesspoint_workflow_manager.py:2599]:
```
    each_result = {
        "changed": self.result["changed"],
        "response": self.result["response"].get("accesspoints_verify"),
        "ap_update_msg": self.result["ap_update_msg"]
    }

```

File: plugins/modules/accesspoint_workflow_manager.py [hunk 2]
```diff
@@ -2584,6 +2586,32 @@ def camel_to_snake_case(self, config):
...
+                       "response": self.result["response"].get("accesspoints_verify"),
+                       "ap_update_msg": self.result["ap_update_msg"]}
+        self.payload["consolidated_result"].append(each_result)
+        self.log("Each execution Result {0}".format(self.pprint(self.result)))
```
Comment (madhansansel) [inline plugins/modules/accesspoint_workflow_manager.py:2603]: Log the message after changing the result.. 
    self.result['changed'] = False
    self.log("Each execution Result {0}".format(self.pprint(self.result)))

File: plugins/modules/accesspoint_workflow_manager.py [hunk 3]
```diff
@@ -2584,6 +2586,32 @@ def camel_to_snake_case(self, config):
...
+        self.result['changed'] = False
+        for each_cosolidated in self.payload["consolidated_result"]:
+            if each_cosolidated['changed']:
+                self.result['changed'] = True
+                break
+        if self.result['changed']:
```
Comment (madhansansel) [inline plugins/modules/accesspoint_workflow_manager.py:2609]: Can you please add a new line before if

File: plugins/modules/accesspoint_workflow_manager.py [hunk 4]
```diff
@@ -2584,6 +2586,32 @@ def camel_to_snake_case(self, config):
...
+            self.status = "success"
+        self.msg = self.pprint(self.payload["consolidated_result"])
```
Comment (madhansansel) [inline plugins/modules/accesspoint_workflow_manager.py:2611]: Add a new line before

File: plugins/modules/accesspoint_workflow_manager.py [hunk 5]
```diff
@@ -2631,8 +2659,9 @@ def main():
         ccc_network.get_diff_state_apply[state](config).check_return_status()
 
         if config_verify:
-            time.sleep(20)
+            time.sleep(10)
```
Comment (madhansansel) [inline plugins/modules/accesspoint_workflow_manager.py:2662]: Add a log message before this sleep..

File: plugins/module_utils/dnac.py [hunk 6]
```diff
@@ -1014,8 +1014,11 @@ def validate_int(item, param_spec, param_name, invalid_params):
...
         }
     """
-
-    item = validation.check_type_int(item)
+    try:
+        item = validation.check_type_int(item)
+    except TypeError as e:
+        invalid_params.append("{0}: value: {1} {2}".format(param_name, item, str(e)))
+        return item
     min_value = 1
```
Comment (madhansansel) [inline plugins/module_utils/dnac.py:1022]: Add a new line after except block.. can we make it simple?

    min_value = param_spec.get("range_min", 1)
    if param_spec.get("range_max") and not (min_value <= item <= param_spec["range_max"]):
        invalid_params.append(
            "{0}: {1} : The item exceeds the allowed range of min: {2} and max: {3}".format(
                param_name, item, param_spec.get("range_min"), param_spec.get("range_max"))
        )

    return item

File: plugins/module_utils/dnac.py [hunk 7]
```diff
@@ -1014,8 +1014,11 @@ def validate_int(item, param_spec, param_name, invalid_params):
             "range_max": 100    # Optional: maximum allowed value
         }
     """
```
Comment (madhansansel) [inline plugins/module_utils/dnac.py:1016]: Can we rename the API to  'validate_integer_within_range' or 'validate_integer_constraints'

**Review by md-rafeek — COMMENTED** [review]
Addressed all 8 comments.

**Review by madhansansel — APPROVED** [review]

**Review by madhansansel — APPROVED** [review]

**Pull request merged**